frontend/common/src/hooks/useLogger.ts

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-console */
+
+import { SeverityLevel, levelIncludes } from "../helpers/loggingUtils";
+import { getLoggingLevel } from "../helpers/runtimeVariable";
+import { reactPlugin } from "../components/AppInsights/ReactPlugin";
+
+export interface Logger {
+  emergency: (message: string) => void;
+  alert: (message: string) => void;
+  critical: (message: string) => void;
+  error: (message: string) => void;
+  warning: (message: string) => void;
+  notice: (message: string) => void;
+  info: (message: string) => void;
+  debug: (message: string) => void;
+}
+// A simple logger for logging to the browser console.
+// Mostly useful during development.
+// Controlled by setting the LOG_CONSOLE_LEVEL environment variable.
+const consoleLoggerLevel = getLoggingLevel("LOG_CONSOLE_LEVEL");
+const consoleLogger: Logger = {
+  emergency: (message: string) => {
+    if (levelIncludes(consoleLoggerLevel, SeverityLevel.Emergency))
+      console.error(message);
+  },
+  alert: (message: string) => {
+    if (levelIncludes(consoleLoggerLevel, SeverityLevel.Alert))
+      console.error(message);
+  },
+  critical: (message: string) => {
+    if (levelIncludes(consoleLoggerLevel, SeverityLevel.Critical))
+      console.error(message);
+  },
+  error: (message: string) => {
+    if (levelIncludes(consoleLoggerLevel, SeverityLevel.Error))
+      console.error(message);
+  },
+  warning: (message: string) => {
+    if (levelIncludes(consoleLoggerLevel, SeverityLevel.Warning))
+      console.warn(message);
+  },
+  notice: (message: string) => {
+    if (levelIncludes(consoleLoggerLevel, SeverityLevel.Notice))
+      console.info(message);
+  },
+  info: (message: string) => {
+    if (levelIncludes(consoleLoggerLevel, SeverityLevel.Info))
+      console.info(message);
+  },
+  debug: (message: string) => {
+    if (levelIncludes(consoleLoggerLevel, SeverityLevel.Debug))
```

Comment:
I couldn't get this debug message in console

Reply:
Hmm, weird.  The console debug level works OK for me.
![image](https://user-images.githubusercontent.com/8978655/211844526-4aac439d-66c0-46f7-b678-c05e49169e52.png)
Could you check in your .env file that LOG_CONSOLE_LEVEL is set to debug and it is passed in your config.js file?
![image](https://user-images.githubusercontent.com/8978655/211844674-16b7d495-c151-4bc0-b078-8019d41b23bc.png)
![image](https://user-images.githubusercontent.com/8978655/211845020-d24c0fdd-c02b-4aa3-bb5d-d32813cbf0c9.png)

Reply:
yeah ..thats what I have.  But only debug button didn't log. 
<img width="784" alt="Screen Shot 2023-01-11 at 10 35 11 AM" src="https://user-images.githubusercontent.com/89482910/211848463-6c9a7716-1072-424b-b559-b425f4849a26.png">

Reply:
oh .. I found the issue .. its my side .. I had settings on my logging level on my console 
<img width="198" alt="Screen Shot 2023-01-11 at 10 45 24 AM" src="https://user-images.githubusercontent.com/89482910/211850990-09fe1371-62c5-492f-8ab0-47593e6bc470.png">

Reply:
I needed to tick that verbose to see this debug message ..